src/anchoredinterval.jl

```diff
@@ -136,37 +135,37 @@ span(interval::AnchoredInterval{P}) where P = abs(P)
 
 ##### CONVERSION #####
 
-function Base.convert(::Type{Interval}, interval::AnchoredInterval{P, T}) where {P, T}
-    return Interval{T}(first(interval), last(interval), inclusivity(interval))
+function Base.convert(::Type{Interval}, interval::AnchoredInterval{P,T,L,R}) where {P,T,L,R}
+    return Interval{T,L,R}(first(interval), last(interval))
 end
 
-function Base.convert(::Type{Interval{T}}, interval::AnchoredInterval{P, T}) where {P, T}
-    return Interval{T}(first(interval), last(interval), inclusivity(interval))
+function Base.convert(::Type{Interval{T}}, interval::AnchoredInterval{P,T,L,R}) where {P,T,L,R}
```

Comment:
Is there a situation where this function is used over the one above? or more so, would there ever be a difference?

Reply:
The signatures of these two functions is identical with the updated `AnchoredInterval` type.

Reply:
Do we need them both then?

Reply:
I misinterpreted your original question. The actual answer to your initial question is that there is a difference between the two functions. One is called as `convert(Interval, ...)` which is just "convert this to a subtype of Interval" where `convert(Interval{Int64}, ...)` is "convert this to a subtype of `Interval` where the element type is `Int64`".

We could combine these so the signature is:
```
Base.convert(::Union{Type{Interval}, Type{Interval{T}}, ...)
```
I may make that change in another PR.

Reply:
I think this is just me trying to wrap my head around multiple dispatch for certain things. I get that one is just strictly `Interval` and the other is `Interval{T}`, but doesn't the `Interval{T}` one stipulate that the `T` has to be the same type as the `T` in the `AnchoredInterval{P,T,L,R}`?, and therefore you wouldn't be able to do something like `convert(Interval{Int64})` if the `T` in the `AnchoredInterval` was a `Float64` per se? Perhaps I am misunderstanding, and both `T`'s don't actually need to match?